src/server/server_family.cc

```diff
@@ -1107,6 +1107,8 @@ void PrintPrometheusMetrics(const Metrics& m, DflyCmd* dfly_cmd, StringResponse*
                            << sdata_res.error().message();
   }
   AppendMetricWithoutLabels("tls_bytes", "", m.tls_bytes, MetricType::GAUGE, &resp->body());
+  AppendMetricWithoutLabels("serialization_bytes", "", m.serialization_bytes, MetricType::GAUGE,
```

Comment:
is it full sync serialization_bytes ? stable state as well?

Reply:
nope, this is only `SliceSnapshot`, i.e. only full sync and *not* stable sync

Reply:
ok, let's maybe call it "full_sync_serialization_bytes" here and in `/metrics` ?

Reply:
The thing is - `SliceSnapshot` is also used by `SAVE`, so `full_sync_*` might be a bit misleading

Reply:
Done :)